admin/src/templates/message_queue_tab.html

```diff
@@ -0,0 +1,46 @@
+<p ng-show="error" class="alert alert-danger" role="alert" translate>admin.message.queue.error</p>
+
+<div class="row flex-row selection-heading">
+  <div class="col col-fixed" translate>admin.message.queue.recipient</div>
+  <div class="col col-fixed" translate>admin.message.queue.created</div>
+  <div class="col col-grow" translate>admin.message.queue.message</div>
+  <div class="col col-fixed" translate>admin.message.queue.status</div>
+  <div class="col col-fixed" translate>admin.message.queue.due</div>
+  <div ng-if="displayLastUpdated" class="col col-fixed" translate>admin.message.queue.updated</div>
+</div>
+
+<div class="loader" ng-show="loading"></div>
+
+<div class="row flex-row selection" ng-repeat="message in messages">
+
+  <div class="col col-fixed">{{message.recipient}}</div>
+
+  <div class="col col-fixed">
+    <relative-date date="message.record.reportedDate" date-format="dateFormat"></relative-date>
+    <div ng-if="message.link">
+      <a href=" ../../medic/_rewrite/#/reports/{{message.record.id}}" target="_blank">
+        <span translate>admin.message.queue.view.report</span>
+      </a>
+    </div>
+  </div>
+
+  <div class="col col-grow">
+    <p ng-if="message.task"><strong>{{message.task}}</strong></p>
+    <span>{{message.content}}</span>
+  </div>
+
+  <div class="col col-fixed">
+    <span class="message-state state-{{message.state}}" ng-class="{ delayed: message.delayed }">{{message.state}}</span>
```

Comment:
Concatenating styles make them really hard to find in an editor. Instead can you just include the state and change the css rule to `.message-state.scheduled`, etc. That should be specific enough.

Reply:
Also we should translate the `message.state` - we already have translations for these.

Reply:
Sure,  only problem is that the states translation, at least in English, add a `timestamp` suffix (eg: https://github.com/medic/medic-webapp/blob/ce12674bb3961d91471bc1b65c35e54de58172ae/ddocs/medic/_attachments/translations/messages-en.properties#L65) . Do you think we should just display them like that or add a separate translation just for the state name?

Reply:
There's a version which doesn't have "timestamp" at the end, eg: `state.pending = pending`: https://github.com/medic/medic-webapp/blob/ce12674bb3961d91471bc1b65c35e54de58172ae/ddocs/medic/_attachments/translations/messages-en.properties#L114